extensions/vscode-vue-language-features/src/features/tsVersion.ts

```diff
@@ -158,6 +149,17 @@ function getWorkspaceTsPaths(useDefault = false) {
 }
 
 function getVscodeTsPaths() {
+	const nightly = vscode.extensions.getExtension('ms-vscode.vscode-typescript-next');
```

Comment:
It seems that user can only select nightly version when user install `ms-vscode.vscode-typescript-next` extension. They can't select the builtin typescript version.

Reply:
It's for consistent with VSCode behavior, because seems VSCode cannot select it too.